tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -39,6 +39,11 @@ SDW_DMIC_STREAM=SDW0-Capture"
 SDW_SPK_STREAM=Playback-SmartAmp,SDW_SPK_IN_STREAM=Capture-SmartAmp,\
 SDW_DMIC_STREAM=Capture-SmartMic,SDW_JACK_OUT_STREAM=Playback-SimpleJack,\
 SDW_JACK_IN_STREAM=Capture-SimpleJack"
+
+# Jack codec + SmartAmp topology. No SDW_DMIC connection
+"cavs-sdw\;sof-mtl-rt713-l0-rt1316-l12\;PLATFORM=mtl,NUM_SDW_AMP_LINKS=2,\
+SDW_SPK_STREAM=Playback-SmartAmp,SDW_SPK_IN_STREAM=Capture-SmartAmp,\
+SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
```

Comment:
@macchian We should use the default stream name. Please see https://github.com/thesofproject/sof/pull/8305 for your reference.

Reply:
@bardliao , Thanks for pointing out!